packages/ckeditor5-utils/src/eventinfo.ts

```diff
@@ -14,11 +14,18 @@ import spy from './spy';
  * manipulate it.
  */
 export default class EventInfo {
+	readonly source: object;
```

Comment:
Should we be explicit about access modifiers and mark these properties with public keyword? It would be consistent with protected/private properties so we don't have a case where private and protected properties have access modifier keyword, but public ones don't.